src/test/java/org/lmdbjava/DbiTest.java

```diff
@@ -146,6 +147,22 @@ public void drop() {
   }
 
   @Test
+  public void dropAndDelete() {
+    final Dbi<ByteBuffer> db = env.openDbi(DB_1, MDB_CREATE);
+    final Dbi<ByteBuffer> nameDb = env.openDbi((byte[]) null);
```

Comment:
It will drop the content in nameDb. The MAIN_DB (nameDb) will never get deleted.
https://github.com/LMDB/lmdb/blob/14cff072ec29f48093a9d40cc79934cf5376af4a/libraries/liblmdb/mdb.c#L9949-L9969